src/carter-mc-barter-narter-is-dope.css

```diff
@@ -1,7 +1,32 @@
-body {
-  min-height: 100vh;
+div{
+  min-height: 5vh;
   display: flex;
   flex-direction: row;
   align-items: center;
   justify-content: center;
 }
```

Comment:
Off we go into _code style comments_! We'll want to avoid the necessity of these, eventually, by implementing a linter... but for now let's get some reps in to feel the pain so we understand why linters are _freakin' sweet_.

How about we agree to include spaces between CSS selector declarations? ie
```
.foo {
  ...stuff
}

.bar {
  ...other stuff
}